source/documentation/breaking-changes/random-with-units.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+title: "Breaking Change: Random With Units"
+introduction: >
+  Sass `random($limit)` will no longer ignore `$limit` units in a future
+  version. Passing a `$limit` with units will be deprecated to ensure the
```

Comment:
```suggestion
  version. Passing a `$limit` with units is deprecated to ensure the
```

This will be landing approximately concurrently with the deprecation warning, so we should use present tense. Same goes elsewhere in the documentation.